_data/simple-icons.json

```
@@ -11017,6 +11017,11 @@
             "source": "https://en.facebookbrand.com/oculus/assets/oculus?audience=oculus-landing",
             "guidelines": "https://en.facebookbrand.com/oculus/"
         },
+        {
+            "title": "Odin",
+            "hex": "1E5085",
```

```suggestion
            "hex": "3882D2",
```
My apologies, I actually think the hex code should be this based on everything except the header of https://odin-lang.org/, and all the colors on https://pkg.odin-lang.org/.